website/docs/r/registry_module.html.markdown

```diff
@@ -77,14 +77,21 @@ resource "tfe_organization" "test-organization" {
   email = "admin@company.com"
 }
 
-resource "tfe_registry_module" "test-no-code-provisioning-registry-module" {
+resource "tfe_registry_module" "test-ncp-registry-module" {
```

Comment:
It might be difficult to remember what `ncp` stands for while using the doc, I suggest we leave the full naming `"test-no-code-provisioning-registry-module"` as before or remove `ncp` totally.